### PR TITLE
[Keyboard] Add NCR-80

### PR DESCRIPTION
**Pull request by j4ckofalltrades**
## Description

Add NCR-80.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by drashna — APPROVED** [review]

File: keyboards/mt/ncr80/hotswap/hotswap.h [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
+
+#define XXX KC_NO
+
+
+#define LAYOUT_tkl_ansi_wkl( \
```
Comment (fauxpark) [inline keyboards/mt/ncr80/hotswap/hotswap.h:23]: Missing print screen, scroll lock and pause/break, so this is not a TKL ANSI WKL layout.
```suggestion
#define LAYOUT( \
```

File: keyboards/mt/ncr80/hotswap/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT_tkl_ansi_wkl(
```
Comment (fauxpark) [inline keyboards/mt/ncr80/hotswap/keymaps/default/keymap.c:20]:
```suggestion
  [0] = LAYOUT(
```

File: keyboards/mt/ncr80/hotswap/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [0] = LAYOUT_tkl_ansi_wkl(
```
Comment (fauxpark) [inline keyboards/mt/ncr80/hotswap/keymaps/via/keymap.c:20]:
```suggestion
  [0] = LAYOUT(
```

File: keyboards/mt/ncr80/hotswap/keymaps/via/keymap.c [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    KC_ESC,  KC_F1,   KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,   KC_F7,   KC_F8,   KC_F9,    KC_F10,     KC_F11,     KC_F12,
+    KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,    KC_6,    KC_7,    KC_8,    KC_9,    KC_0,     KC_MINS,    KC_EQL,     KC_BSPC,                 KC_INS,  KC_HOME, KC_PGUP,
+    KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,    KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,     KC_LBRC,    KC_RBRC,    KC_BSLS,                 KC_DEL,  KC_END,  KC_PGDN,
+    KC_CAPS, KC_A,    KC_S,    KC_D,    KC_F,    KC_G,    KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN,  KC_QUOT,                KC_ENT,
+    KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,    KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH,  KC_RSFT,                                                  KC_UP,
+    KC_LCTL, KC_LALT,                   KC_SPC,                                      KC_RALT,           KC_RCTL,                                         KC_LEFT, KC_DOWN, KC_RGHT),
+  [1] = LAYOUT_tkl_ansi_wkl(
```
Comment (fauxpark) [inline keyboards/mt/ncr80/hotswap/keymaps/via/keymap.c:27]:
```suggestion
  [1] = LAYOUT(
```

File: keyboards/mt/ncr80/hotswap/keymaps/via/keymap.c [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    _______,  _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,    _______,     _______,     _______,
+    _______,  _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,     _______,    _______,     _______,                 _______,  _______, _______,
+    _______,  _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,     _______,    _______,    _______,                 _______,  _______,  _______,
+    _______, _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,  _______,                _______,
+    _______, _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______, _______,  _______,  _______,                                                  _______,
+    _______, _______,                   _______,                                      _______,           _______,                                         _______, _______, _______),
+  [2] = LAYOUT_tkl_ansi_wkl(
```
Comment (fauxpark) [inline keyboards/mt/ncr80/hotswap/keymaps/via/keymap.c:34]:
```suggestion
  [2] = LAYOUT(
```

File: keyboards/mt/ncr80/hotswap/keymaps/via/keymap.c [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    _______,  _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,   _______,    _______,     _______,     _______,
+    _______,  _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,     _______,    _______,     _______,                 _______,  _______, _______,
+    _______,  _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,     _______,    _______,    _______,                 _______,  _______,  _______,
+    _______, _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______,  _______,                _______,
+    _______, _______,    _______,    _______,    _______,    _______,    _______,    _______,    _______, _______,  _______,  _______,                                                  _______,
+    _______, _______,                   _______,                                      _______,           _______,                                         _______, _______, _______),
+  [3] = LAYOUT_tkl_ansi_wkl(
```
Comment (fauxpark) [inline keyboards/mt/ncr80/hotswap/keymaps/via/keymap.c:41]:
```suggestion
  [3] = LAYOUT(
```

File: keyboards/mt/ncr80/hotswap/info.json [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+    "manufacturer": "NCR",
+    "url": "https://www.aliexpress.com/item/1005003345941543.html",
+    "maintainer": "qmk",
+    "usb": {
+        "vid": "0x4D54",
+        "pid": "0x2002",
+        "device_version": "0.0.1"
+    },
+    "layouts": {
+          "LAYOUT_tkl_ansi_wkl": {
```
Comment (fauxpark) [inline keyboards/mt/ncr80/hotswap/info.json:12]:
```suggestion
          "LAYOUT": {
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by j4ckofalltrades** [comment]
@lesshonor 

I'm not certain but I think it should, my understanding is that the only difference from R1 vs R2 is that R2 has a different shade of grey and beige for the plastic housing.

**Pull request merged**